.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and test
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+# This allows a subsequently queued workflow run to interrupt and cancel previous runs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Run-Unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [ 3.8, 3.9, '3.10' ]
```

Comment:
```suggestion
        python: [ 3.7, 3.8, 3.9, '3.10' ]
```

Reply:
@kaxil The tests dont run on 3.7, so we shouldnt add it as per my understanding